pdf_parser/pdf_parse.py

```diff
@@ -20,25 +20,26 @@ def parse_pdf_document(document):
     cmd = [
             'pdftohtml',
             '-i',
-            '-q',
             '-xml',
             document.name,
             parsed_path
             ]
 
     try:
         with open(os.devnull, 'w') as FNULL:
-            subprocess.check_call(cmd, stdout=FNULL)
+            subprocess.check_call(cmd, stdout=FNULL, stderr=FNULL)
     except subprocess.CalledProcessError as e:
         logger.warning(
             "The pdf [%s] could not be converted: %r",
             document.name,
             e.stderr,
         )
-        return None
-
-    html_file = open(parsed_path, 'rb')
-    soup = bs(html_file.read(), 'html.parser')
+        return None, None
+    try:
+        html_file = open(parsed_path, 'rb')
```

Comment:
Do we close this file somewhere, or would a context manager be more appropriate here?

Reply:
It's closed at the end of the function. It could be in a context manager if you think it's better

Reply:
Usually best to close your file descriptions as soon as you're done with them -- if you had an exception between here and end of the function, you'd end up with an fd leak. But, up to you whether to fix now or later.